server/src/main/scala/za/co/absa/atum/server/api/http/Routes.scala

```diff
@@ -60,6 +62,16 @@ trait Routes extends Endpoints with ServerOptions {
         createOrUpdateAdditionalDataEndpointV2,
         PartitioningController.createOrUpdateAdditionalDataV2
       ),
+      createServerEndpoint[
+        (Long, UUID),
```

Comment:
sometimes types are here and sometimes not, why's that?

Reply:
it's because sometimes compiler needs that defined explicitly as it cannot infer everything ...